features/upgrade/build/build-upgrade.feature

```diff
@@ -0,0 +1,34 @@
+Feature: build related upgrade check
+  # @author wewang@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  Scenario: basic verification for upgrade build test
+    Given I switch to the first user
+    When I run the :new_project client command with:
+      | project_name | devexp-upgrade |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | app_repo     | openshift/ruby~https://github.com/openshift/ruby-ex |
+    Then the step should succeed
+    And the "ruby-ex-1" build completes
+    And a pod becomes ready with labels:
+      | deploymentconfig=ruby-ex |
+    When I run the :new_app client command with:
+      | app_repoe | openshift/ruby~https://github.com/openshift/ruby-hello-world |
+      | strategy  | docker |
+    Then the step should succeed
+    And the "ruby-hello-world-1" build completes
+    And a pod becomes ready with labels:
+      | deploymentconfig=ruby-hello-world |
+
+  # @author wewang@redhat.com
+  # @case_id OCP-13025
+  @upgrade-check
+  @users=upuser1,upuser2
+  Scenario: Check docker and sti build works well after upgrade
+    Given I switch to the first user
+    When I use the "devexp-upgrade" project
+    And status becomes :running of 1 pods labeled:
```

Comment:
It’s better to trigger a new build for two type builds

Reply:
agree

Reply:
Updated, please check it again, thanks